.golangci.yml

```diff
@@ -93,39 +93,70 @@ linters-settings:
 #    allow-separated-leading-comment: false
 
 linters:
-  disable-all: true
   enable:
-    - asciicheck
-    - deadcode
-    - dogsled
-    - errcheck
-    - gocognit
-    - goconst
-    - gocritic
-    - gocyclo
-    - godox
-#    - goerr113
-    - gofmt
-    - golint
-#    - gomnd
-    - gosec
-    - gosimple
-    - govet
-    - ineffassign
-    - maligned
-    - megacheck
-    - misspell
-    - nestif
-    - rowserrcheck
-    - structcheck
-    - staticcheck
-    - typecheck
-    - varcheck
-    - unconvert
-    - unparam
-    - unused
-    - whitespace
-# https://github.com/bombsimon/wsl
-#    - wsl
-
-# TODO: exclude testutil from being analysed
+    - deadcode             # Finds unused code [fast: true, auto-fix: false]
```

Comment:
I noticed that some of the linters have been disabled (`megacheck` and possibly others) and some others have been activated (`goimports` and possibly others). That's worrisome, as we might get new unexpected linter errors from now on. Can we please have the list updated in a separate PR? Also, wouldn't it be better to have them ordered alphabetically? Thanks.

Reply:
The linters that are deactivated are new ones that we haven't used before.
The activated ones are working fine.
On the next PR I'll try to activate the new ones one by one and fix the code accordingly.

Reply:
Note that megacheck is a group alias. All individual checks are listed.